sample/src/main/java/io/getstream/chat/example/utils/UserStorage.java

```diff
@@ -0,0 +1,84 @@
+package io.getstream.chat.example.utils;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import androidx.annotation.Nullable;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.List;
+
+public class UserStorage {
+
+    private static Context context;
+    private static SharedPreferences preferences;
+    private static List<UserConfig>users;
+
+    private static final String USER_DATA_FILENAME = "users-config.json";
+    private static final String CURRENT_USER_ID = "current-user-id";
+    private static final String APP_PREFERENCE = "AppPreferences";
+
+    public static void init(Context mContext) {
+        context = mContext;
+        preferences = context.getSharedPreferences(APP_PREFERENCE, Activity.MODE_PRIVATE);
+        parseJson();
```

Comment:
We should also get list of users from `BuildConfig`. 
That's important to let developers define and use their own users and apiKeys.